dandi/files/bids.py

```diff
@@ -15,10 +15,12 @@
 from ..metadata import add_common_metadata, prepare_metadata
 from ..misctypes import Digest
 
-BIDS_TO_DANDI = {
-    "subject": "subject_id",
-    "session": "session_id",
-}
+BIDS_ASSET_ERRORS = [
```

Comment:
If `BIDS_ASSET_ERRORS` and `BIDS_DATASET_ERRORS` are only going to be used on the right side of `in`, they would be better as sets.